ament_clang_tidy/ament_clang_tidy/main.py

```diff
@@ -77,6 +77,9 @@ def main(argv=sys.argv[1:]):
         '--system-headers',
         action='store_true',
         help='Displays errors from all system headers')
+    parser.add_argument(
+        '--packages-select', nargs='+', metavar='PKG_NAME',
```

Comment:
Can the `metavar` be something like `PKG_NAME1 [PKG_NAME2] ...` (or whatever the convention is)?

Reply:
I copied this from the `colcon` argument `--packages-select` so it has the same form as the `colcon` arg of the same name.